### ISSUE TITLE
Negative time ago shown in recent games after daylight saving time switch

### ISSUE THREAD

**Issue by MatmaRex** [HearthSim/Hearthstone-Deck-Tracker#4508]
### Expected Behavior

Expected the "My Games" section to show the amount of time that passed since I played my recent matches.

### Actual Behavior

The "My Games" section shows that a negative number of minutes passed since I played several recent matches.

### Steps to reproduce behavior

Play some games during the hour when Northern Hemisphere countries move an hour back to switch from summer time to winter time. See <https://en.wikipedia.org/wiki/Daylight_saving_time#Procedure>.

### Log/Screenshots

![deck tracker dst](https://github.com/HearthSim/Hearthstone-Deck-Tracker/assets/160413/040532a0-acc9-4c85-8188-ae32eecfa3a3)

**Comment by Sneaky1000** [comment]
@MatmaRex Were you ever able to resolve this? I changed my system's date and time, forgot to change it before launching HDT, and now the software no longer works unless I set my time back (it shows my collection and deck wins/losses as negative time).

**Comment by MatmaRex** [comment]
Nope, the problem went away by itself as the real world time advanced and the incorrectly saved times became the past. I expect that I'll run into it again this autumn during the next daylight saving time switch.
